en/docs/fhir/guides/overview-of-fhir.md

```diff
@@ -0,0 +1,32 @@
+# Working with FHIR
+
+## Overview
+HL7 FHIR (Fast Healthcare Interoperability Resources) is a standard for exchanging healthcare information electronically. Developed by Health Level Seven International (HL7), FHIR is designed to enable easy, fast, and flexible sharing of healthcare data across different systems, ensuring that patient information is accessible, accurate, and up-to-date.
+
+The key features of the FHIR specification are:
+
+1. **Modular Structure**: It uses a modular approach with "resources" as basic units of information which represent common healthcare entities like patients, medications, appointments, and lab results. These resources can be combined and extended to meet specific needs.
+2. **Interoperability**: FHIR is designed to facilitate interoperability between different healthcare systems. It supports both simple data exchange (like patient demographics) and complex workflows (like care coordination).
+3. **API-based**: It's designed to work with modern web technologies, using RESTful APIs for data exchange. Which means data can be easily retrieved and updated over the web using HTTP requests.
+4. **Flexibility and Extensibility**: FHIR allows for customization to meet specific requirements while maintaining a core set of common elements.
+5. **Support for Multiple Formats**: FHIR supports data representation in JSON, XML, and RDF formats, making it compatible with various systems and easier to work with in different programming environments.
+6. **Security**: FHIR incorporates standard security protocols, including OAuth2 and SMART on FHIR, to ensure that patient data is securely accessed and shared.
+
+WSO2 Healthcare provides a comprehensive set of products/tools to support the FHIR-based digital transformation and compliance needs of any payer, provider, or related business.
+
+???+ note
```

Comment:
Can you put this as a reusable component in md,

Add a file like following

!!! note
This is a reusable note. You can include this note in any document by referencing this file.

Refer in the actual guide

{!includes/note.md!}

Reply:
fixed in 
558e0e6